src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,20 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+    if (typeof text === 'string' || text instanceof String){
```

Comment:
Nice.

But do the reverse early so you don't have to wrap your whole program in a try catch.